public/apps/configuration/panels/role-view/role-view.tsx

```diff
@@ -321,10 +360,14 @@ export function RoleView(props: RoleViewProps) {
         </EuiPageContentHeaderSection>
       </EuiPageContentHeader>
 
-      <EuiTabbedContent tabs={tabs} />
+      <EuiTabbedContent
+        tabs={tabs}
+        initialSelectedTab={props.prevAction === SubAction.mapuser ? tabs[1] : tabs[0]}
```

Comment:
Minor: for better readability, probably you can create a
const MAP_USER_TAB_INDEX = 1;
const PERMISSIONS_TAB_INDEX = 0;

Reply:
Agreed.

Reply:
Created constants for tab index.